kyaml/setters2/list.go

```diff
@@ -192,3 +192,22 @@ func (l *List) count(path, name string) (int, error) {
 
 	return s.Count, err
 }
+
+// CheckRequiredSettersSet iterates through all the setter definitions in openAPI
+// schema and returns error if any of the setter has required field true and isSet false
+func CheckRequiredSettersSet() error {
```

Comment:
nit: Could we put this at least in a separate file? It doesn't seem to really have anything to do with listing setters.

Reply:
+1

Reply:
@mortent Done.